webapi/src/main/scala/org/knora/webapi/routing/VersionRoute.scala

```diff
@@ -32,39 +23,24 @@ trait VersionCheck {
 
   override implicit val timeout: Timeout = 1.second
 
-  protected def createResponse(result: VersionCheckResult): HttpResponse =
+  protected def createResponse(): HttpResponse = {
+    val sipiVersion: String = BuildInfo.sipi.split(":").apply(1)
+    val fusekiVersion: String =  BuildInfo.fuseki.split(":").apply(1)
+
     HttpResponse(
       status = StatusCodes.OK,
       entity = HttpEntity(
         ContentTypes.`application/json`,
         JsObject(
-          "name" -> JsString(result.name),
-          "webapi" -> JsString(result.webapi),
-          "scala" -> JsString(result.scala),
-          "akkaHttp" -> JsString(result.akkaHttp),
-          "sipi" -> JsString(result.sipi),
-          "fuseki" -> JsString(result.fuseki)
-        ).compactPrint
+          "name" -> JsString("version"),
```

Comment:
Do we even need this line?

Reply:
probably not, but then it is a breaking change.

Reply:
Not really. but instead of improvements to this route I just fixed it with small refactoring to avoid breaking changes...